<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Liberation Serif&quot;;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>1281</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QTableWidget" name="m_finalTableWidget">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e0e0e0;
font: 11pt &quot;Liberation Serif&quot;;</string>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Vorname</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nachname</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Geschlecht</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Geburtsdatum</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>OperationDate</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>V-ID</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NotfallContact</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nt</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NeedleTip</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NeedleLength</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NeedleDiameter</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NeedleDepth</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wundposition</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wundtiefe</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wundgröße</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N° Stichpunkte</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>148</red>
         <green>192</green>
         <blue>241</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_delete">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                                color: white;
                                border: 2px solid #555;
                                border-radius: 20px;
                                border-style: outset;
                                background: qradialgradient(
                                    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #888
                                );
                                padding: 5px;
                            }

                            QPushButton:hover {
                                background: qradialgradient(
                                    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                );
                            }

                            QPushButton:pressed {
                                border-style: inset;
                                background: qradialgradient(
                                    cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                );
                            }</string>
      </property>
      <property name="text">
       <string>Löschen</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="flabel_nDiameter_2">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>© 2023 AS Interactive HealthTech. Alle Rechte vorbehalten.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_backLast">
      <property name="font">
       <font>
        <family>Liberation Serif</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                                color: white;
                                border: 2px solid #555;
                                border-radius: 20px;
                                border-style: outset;
                                background: qradialgradient(
                                    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #888
                                );
                                padding: 5px;
                            }

                            QPushButton:hover {
                                background: qradialgradient(
                                    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                );
                            }

                            QPushButton:pressed {
                                border-style: inset;
                                background: qradialgradient(
                                    cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                    radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                );
                            }</string>
      </property>
      <property name="text">
       <string>Zurück</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
